docs/service/community-registry.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Community registry
+---
+
+The Community registry adds to the Wave containerization lifecycle a regular container registry to host image builds permanently and is accessible publicly by anyone.
+
+The community registry is built using [Docker Distribution][docker] and hosted on AWS infrastructure. Images are cached and served via Cloudflare CDN.
+
+*WIP Diagram*
```

Comment:
Who are we waiting on for this?

Reply:
@pditommaso and @munishchouhan would probably need to approve one of the diagram choices in the Google Doc, then Design will know which diagram that we'd like created.